.changes/badge_count.md

```diff
@@ -0,0 +1,5 @@
+---
+"tao": patch
+---
+
+Add **set_badge_count** for Linux, iOS; **set_badge_label** for Macos; **set_overlay_icon** for Windows
```

Comment:
I prefer to list each method like this
```suggestion
Add `WindowExtUnix::set_badge_count`, `WindowExtWindow::set_overlay_icon`
```
and so on for the rest of added methods